components/brave_vpn/brave_vpn_service.cc

```diff
@@ -1013,8 +1013,8 @@ void BraveVpnService::OnCredentialSummary(const std::string& domain,
           base::BindOnce(&BraveVpnService::OnPrepareCredentialsPresentation,
                          base::Unretained(this), domain));
     } else {
-      VLOG(1) << __func__ << " : Credential appears to be expired.";
-      SetPurchasedState(env, PurchasedState::EXPIRED);
+      VLOG(1) << __func__ << " : Credential is not active.";
```

Comment:
My only concern with this change is that now I don't think anything will use the expired status (except a unit test; but maybe I missed something). If you have VPN and then billing fails for some reason OR if the user cancels it the intention was to have it show expired

Having it show the not purchased might be OK because the person can just buy it again (on accounts website)

Reply:
Im working on expired state in another issue

Reply:
https://github.com/brave/brave-core/pull/15463